src/components/selector-dropdown/SelectorDropdown.js

```diff
@@ -59,8 +61,17 @@ class SelectorDropdown extends React.Component<Props, State> {
     }
 
     componentWillReceiveProps(nextProps: Props) {
```

Comment:
While you are at it, can you rename this to UNSAFE_componentWillReceiveProps. Or if you want to be adventurous, change this function to be componentDidMount :)